.github/workflows/depenabot_auto_merge.yml

```diff
@@ -1,5 +1,5 @@
 name: Dependabot Auto Merge
-on: [pull_request, push]
```

Comment:
Im not sure about this one? Did you check the docu?

Reply:
sorry I thought I updated this. pull_request should be enough

Reply:
I wasn’t sure. Glad you checked 👍